brevbaker-dsl/src/test/kotlin/no/nav/pensjon/brev/template/render/dsl/SomeDtoSelectors.kt

```diff
@@ -0,0 +1,87 @@
+package no.nav.pensjon.brev.template.render.dsl
+
+import no.nav.pensjon.brev.template.Expression
+import no.nav.pensjon.brev.template.TemplateModelSelector
+import no.nav.pensjon.brev.template.UnaryOperation
+import no.nav.pensjon.brev.template.dsl.TemplateGlobalScope
+
+object SomeDtoSelectors {
```

Comment:
Jeg antar at grunnen til at du legger til selectors her er fordi vi ikke har template-model-generator tilgjengelig i brevbaker-dsl modulen? Tester vi "for mye" her? Burde vi se på å omformulere testene slik at de ikke er avhengig av selectors? Eller gir det mening å ha masse selectors committed?

Reply:
Evt. kan vi vurdere å ha template-model-generator som en test avhengighet?

Reply:
`template-model-generator` er avhengig av `brevbaker-dsl`, så trur vi fort blir sirkulært avhengige ved å introdusere avhengnad motsett veg.

Trur det er fint å få bort selectors frå desse testane, men det er ikkje heilt rett fram.